packages/x-components/src/x-modules/tagging/store/emitters.ts

```diff
@@ -7,5 +7,6 @@ import { taggingXStoreModule } from './module';
  * @internal
  */
 export const taggingEmitters = createStoreEmitters(taggingXStoreModule, {
-  ConsentChanged: state => state.consent!
+  ConsentChanged: state => state.consent!,
+  SearchTaggingEmitted: state => state.queryTaggingInfo!
```

Comment:
I know is not related to the PR but we need to avoid import from barrels (line 1)

```javascript
import { createStoreEmitters } from '../../../store/utils/store-emitters.utils';
```